src/test/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileRepositorySpec.scala

```diff
@@ -16,8 +16,9 @@ class FileRepositorySpec extends AnyFlatSpec with ScalaFutures with Matchers {
     val db = DbConnection.db
     val fileRepository = new FileRepository(db)
     val consignmentId = UUID.fromString("c03fd4be-58c1-4cee-8d3c-d162bb4f7c02")
+    val seriesId = UUID.fromString("1436ad43-73a2-4489-a774-85fa95daff32")
 
-    TestUtils.createConsignment(consignmentId, userId)
+    TestUtils.createConsignment(consignmentId, seriesId, userId)
```

Comment:
One way to avoid having to update all of these existing tests is to give the `seriesId` parameter a default value in the `createConsignment` method. Then the tests in `FileRepositorySpec` could stay the same, and the new tests could pass a specific value.

Reply:
Ah yes, that is a good idea; I completely forgot you could do this in Scala :smile:

Reply:
Is this change still to be done?